config.rb

```diff
@@ -34,6 +34,26 @@
   end
 end
 
+# Conference Lists
+data.talks.each do |city, years|
+  years.each do |year, talks|
+    conf_talks = talks.map do |slug, talk|
+      talk.tap {|t| t[:slug] = slug}
+    end
+
+    proxy "talks/#{city}/#{year}", "talk-list.html", locals: { talks: conf_talks, city: city, year: year }, ignore: true
```

Comment:
Added links to all the talks for a conference. This was mostly just to make it easy to check all the 2020 talks, but it could be a nice link to share on twitter after a conference or something.